DEVELOPER_GUIDE.md

```diff
@@ -466,25 +463,25 @@ Concurrent requests received for the handler will be handled by a thread pool co
     </StackObjectPool>
 ```
 
-## Encrypting Passwords with Cipher tool
-The configuration file contains both ASE access token and the password for the proxy server. If the customer wants to encrypt these fields, that capability is provided with the Cipher tool.
+## Encrypting passwords with the Cipher Tool
```

Comment:
You may need to change the link in line no 154.

Reply:
Fixed